presto-product-tests/conf/tempto/tempto-configuration.yaml

```diff
@@ -3,3 +3,23 @@ databases:
     host: hadoop-master
   presto:
     host: presto-master
+
+  prestoConfigUserAlice:
```

Comment:
Do you mean, change the names of the presto connections to `alice@presto` and `bob@presto` ?